assignments/arrays.js

```diff
@@ -63,32 +63,45 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+// console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let filteredArray = invetory.filter((car)) => {
+    if (car.id === 33){
+        return true;
+    }
+    return false;
```

Comment:
This is all extra. 

`.filter` works exactly like `.map` in that it returns a new array after the code is executed. Where it is different is that the function written should be used as a filter, because the returned array only adds the item to it if it meets the criteria. It would be written as such:
``` JS
const car33 = inventory.filter(car => car.id == 33);
// This will return => [ {car33 obj} ]
```
You can then work with that.

I would recommend having done this with a for loop if you weren't sure how to do it with `filter`. Especially considering it wasn't part of lecture.